packages/autocomplete/src/parse.ts

```diff
@@ -60,15 +63,26 @@ export function parseAutocomplete(template: string, theme: any = {}, extraShorth
       input,
       /\$([\w.|]+)/g,
       (m) => {
-        parts.push({
-          type: 'theme',
-          objects: m[1].split('|').map((i) => {
+        const objects = m[1].split('|').map((i) => {
+          try {
             return i.split('.').reduce((v, k) => {
               if (!k || !v[k])
                 throw new Error(`Invalid theme key ${k}`)
               return v[k]
             }, theme)
-          }),
+          }
+          catch (e) {
+            if (isVSCode)
+              vscodeErrors.add(String(e))
+            else
+              throw e
+
+            return null
+          }
+        }).filter(Boolean)
+        parts.push({
+          type: 'theme',
+          objects,
```

Comment:
Same as it, restore it.

Reply:
Do you have any good ideas? Because the try catch in the vscode environment cannot intercept the error here.